app/models/template_section_block.rb

```diff
@@ -4,31 +4,4 @@ class TemplateSectionBlock < ApplicationRecord
   has_one :requirement_template, through: :requirement_template_section
 
   acts_as_list scope: :requirement_template_section, top_of_list: 0
-
-  validate :validate_block_uniqueness_on_template
-
-  private
-
-  def validate_block_uniqueness_on_template
-    return unless duplicate_requirement_block_on_template_exists?
-
-    errors.add(
-      :requirement_block,
-      message:
-        I18n.t(
-          "model_validation.template_section_block.requirement_block_on_template_exits",
-          requirement_block_name: requirement_block.name,
-        ),
-    )
-  end
-
-  def duplicate_requirement_block_on_template_exists?
-    requirement_template
-      .requirement_template_sections
-      .joins(:template_section_blocks)
-      .where(template_section_blocks: { requirement_block: requirement_block })
-      .where.not(template_section_blocks: { id: id })
-      .limit(1)
-      .exists?
-  end
```

Comment:
Is this validation replicated somewhere else, if not then is this constraint not required anymore?

Reply:
I'm not sure why this was removed, I'll put it back

Reply:
nevermind, this was moved on purpose to requirement_template.rb in 
https://github.com/bcgov/HOUS-permit-portal/commit/27ce318359d11c04444f59509a29e76c090d1d36